app/src/main/java/com/hammer/talkbbokki/presentation/detail/DetailScreen.kt

```diff
@@ -85,16 +67,12 @@ fun DetailScreen(
     Column(
         modifier = modifier
             .fillMaxSize()
-            .background(CategoryLevelDummy.valueOf(level).backgroundColor)
+            .background(TopicLevel.valueOf(level).backgroundColor)
     ) {
         DetailHeader(cardFace = cardFace, onBackClick = { cardFace = CardFace.BACK })
         Box(modifier = modifier.fillMaxSize()) {
             DetailFlipCard(
-                Modifier.align(Alignment.Center),
-                cardFace,
-                onClickToList,
-                item,
-                onClickBookmark
+                Modifier.align(Alignment.Center), cardFace, item, onClickToList, onClickBookmark
```

Comment:
🚫 **[ktlint]** <argument-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Argument should be on a separate line (unless all arguments can fit a single line)